week2/index.js

```diff
@@ -0,0 +1,129 @@
+"use strict";
+
+const fs = require("fs");
+
+const todoFile = "todos.txt";
+const helpFile = "help.txt";
+
+const command = process.argv.slice(2);
+let tasks = "";
+
+
+function readTodoFile() {
+
+    fs.readFile(todoFile, "utf8", (err, data) => {
+
+        if (err) {
+            handleError(todoFile, err);
+        }
+        tasks = tasks.concat(data);
+
+        checkCommand();
+    });
+}  
+
+
+function writeTodoFile() {
+
+    fs.writeFile(todoFile, tasks, (err) => {
+
+        if (err) {
+            handleError(todoFile, err);
+        }
+        console.log("Successfully made changes to list ");
+    });  
+}
+
+
+function displayHelp() {
+
+    fs.readFile(helpFile, "utf8", (err, data) => {
+
+        if (err) {
+            handleError(helpFile, err);
+        }
+        console.log(data);
+    });
+}
+
+
+
+function handleError(file, err) {
+
+    if (err.code === "ENOENT") {
+        console.log(`${file} does not exist`);
+
+        if (file === todoFile) {
```

Comment:
This could be refactored as `if (err.code === "ENOENT" && file === todoFile)`, since you don't do anything with the `else` cases of either.